src/styles/brackets.less

```diff
@@ -1110,16 +1091,15 @@ a, img {
         content: " ";
         left: 0;
         z-index: @z-index-brackets-inline-editor-shadow;
-    }
-
-    .shadow.top {
-        top: 0px;
-        background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0));
-    }
-
-    .shadow.bottom {
-        bottom: 0px;
-        background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.1));
+        
+        &.top {
+            top: 0;
+            background-image: -webkit-linear-gradient(top, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0));
```

Comment:
I've already searched for all occurences of old/prefixed syntax that we can replace more or less easily. As they are in different files, I'd like to have them all replaced in a new PR after this one (which focuses on `brackets.less`) is merged.